packages/jsActions/mobile-resources-native/package.json

```diff
@@ -36,7 +36,7 @@
     "react-native-sound": "0.11.0",
     "react-native-touch-id": "4.4.1",
     "url-parse": "^1.4.7",
-    "react-native-permissions": "3.0.4",
+    "react-native-permissions": "3.3.1",
```

Comment:
I suggest aligning native template, which is currently [3.0.4](https://github.com/mendix/native-template/blob/release/6.2.0/package.json#L26)

Reply:
done 
https://github.com/mendix/native-template/pull/540